content/docs/tech-insights/track-dependency-versions/index.md

```diff
@@ -0,0 +1,273 @@
+
+3. Create a compound check:
+    1. Set **Type** to `Or`.
+    2. Set the first condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | GitHub repository file list |
+        | Fact | List of files |
+        | Fact operator | Does not contain |
+        | Value | package.json |
+
+    2. Press **+ADD CONDITION** and set the second condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | Package JSON information |
+        | Fact | React version |
+        | Fact operator | Greater than semantic version |
+        | Value | 16.0.0 |
+    3. Press **+ADD CONDITION** and set the third condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | Package JSON information |
+        | Fact | Has React dependency |
+        | Fact operator | Is False |
+
+    ![Check Condition section fields](./react/React_Check_Condition_Section.png)
+
+4. Press **DRY RUN**
+
+5. (Optional) Add a URL to documentation outlining the steps to update the React version in the component.
+
+    ![Check Fix section displayed](./Check_Fix_Section.png)
+
+6. Use the **Applies to** filter to target this data source at some components which you expect to have React versions. We recommend starting with a highly targeted filter for initial experimentation and iteration. You can widen the filter later to capture more results. 
+
+    ![About Data Source section displayed](./react/React_Applies_To_Section.png)
+
+7. Press **SAVE**.
+
+### Check Results
+
+The results of this check tell us who we need to reach out to to ask for a React version upgrade.
+
+![Check Results displayed](./react/React_Check_Results_Section.png)
+
+- The backstage-demo component passes the check with the version `17.0.59`.
+- The react-native-sample-app component fails the check with the version `15.4.0`
+- The other components passes the check due to not having a React dependency.
+
+Since we already know the owners of these components, it’s easy to reach out and ask them to upgrade their React version.
+
+</details>
+
+<details>
+
+<summary style="font-size: 24px;">.NET Framework Version</summary>
+
+## Record the .NET Framework version automatically
+
+In order to track the .NET Framework versions used in the catalog, we are going to create a Data Source In Roadie Tech Insights. It will be setup to run on a scheduled basis and extract the version specified in the CSProj file.
+
+### CSProj Data Source
+
+1. Visit Tech Insights, select the **Data Sources** tab, and press **ADD DATA SOURCE**.
+
+   ![a button on a web interface](./Add_Data_Source.png)
+
+2. Enter a descriptive **Name** and **Description**.
+
+   ![About section input fields](./dot-net/DotNet_Data_About_Section.png)
+
+3. Set the Data Provider **Type** to `Component Repository File`, **Location** to `{{metadata.name}}.csproj`. Select a value for **Select Entity to test data source against** and press **TEST**.
```

Comment:
Maybe worth mentioning here what the `{{ metadata.name }}` does and possibly hint that other fields from the entity can be used as well, or hardcoded filenames also work which might be a convention in many companies for these build files

Reply:
I'll add a note here upon next commit.